### DISCUSSION TITLE
Code unable to run i recently installed this

### DISCUSSION THREAD

**Discussion by saurabhkaramankar** [cpeditor/cpeditor#1146] [Q&A]
- ``
![Screenshot (419)](https://user-images.githubusercontent.com/70636541/216799674-31cd74e1-fcd1-456f-9533-d45a173d8fc4.png)

**Comment by ouuan** [comment]
You need to install a C++ compiler and add it to PATH. If you don't know how to do, an easy way is to download the asset with "gcc" and "LLVM" in the name, like [cpeditor-6.10.2-windows-x64-setup-with-gcc-11.2.0-LLVM-12.0.1.exe](https://github.com/cpeditor/cpeditor/releases/download/6.10.2/cpeditor-6.10.2-windows-x64-setup-with-gcc-11.2.0-LLVM-12.0.1.exe).

**Comment by saurabhkaramankar** [comment]
i have downloaded asset now but its still showing same problem

**Reply by ouuan** [reply]
Just to make sure: you said you "downloaded asset", I'm assuming you not only downloaded it but also have installed it.

What's your PATH environment variable? Have you installed other compilers or IDEs before?

**Comment by md-shamim-ahmad** [comment]
* Hey you first download & install a compiler for ```c``` & ```c++```.
There have a lot of compiler but [```MinGW```](https://sourceforge.net/projects/mingw/) is good one of them.
* Then copy the bin file location of the compiler which you install.
* Press ```Windows key``` and search ```Environment Variables or System Properties``` and hit ```enter```.
* Then click ```Environment Variables ``` button.
* Then click ```Path``` under the ```System Variables``` and hit ```Edit```.
* Then click ```New ``` and Paste you copied bin file location. 
* Then click ```ok``` button.
Finally your setup will be completed. So close your ```cp editor``` and reopen it and work good :)
